packages/ui/src/lib/dropdown/Dropdown.spec.ts

```diff
@@ -25,56 +25,57 @@ import Dropdown from './Dropdown.svelte';
 import DropdownTest from './DropdownTest.svelte';
 
 test('a button is created', async () => {
-  render(Dropdown);
+  render(Dropdown, { ariaLabel: 'Dropdown' });
 
-  const input = screen.getByRole('button');
+  const input = screen.getByRole('button', { name: 'Dropdown' });
   expect(input).toBeInTheDocument();
 });
 
 test('initial value is visible', async () => {
   render(Dropdown, {
     value: 'a value',
+    ariaLabel: 'Dropdown',
   });
 
-  const input = screen.getByRole('button');
+  const input = screen.getByRole('button', { name: 'Dropdown' });
   expect(input).toBeInTheDocument();
   expect(input).toHaveTextContent('a value');
 });
 
 test('opening dropdown does not submit forms (prevents default button action)', async () => {
-  render(Dropdown);
+  render(Dropdown, { ariaLabel: 'Dropdown' });
 
-  const input = screen.getByRole('button');
+  const input = screen.getByRole('button', { name: 'Dropdown' });
```

Comment:
I don't understand why we need to replace all the getByRole in all the tests ?

Reply:
I thought we wanted to check the accessibility of the button with the aria-label in unit tests.